rubix/core/ifu.py

```diff
@@ -140,14 +168,18 @@ def get_calculate_datacube(config: dict) -> Callable:
     calculate_cube_fn = jax.tree_util.Partial(calculate_cube, num_spaxels=num_spaxels)
     calculate_cube_pmap = jax.pmap(calculate_cube_fn)
 
-    def calculate_datacube(inputs: dict[str, jax.Array]) -> dict[str, jax.Array]:
+    def calculate_datacube(rubixdata: object) -> object:
```

Comment:
how will this work with the type checking? will it overwrite it? we should keep an eye on this :-)

Reply:
No idea, how this works, so we have to be very careful on the branch documentation, because there it is already changed